clang/test/CXX/dcl.dcl/dcl.attr/dcl.attr.nodiscard/p2.cpp

```diff
@@ -171,21 +187,3 @@ void f() {
 #endif
 }
 } // namespace discarded_member_access
-
-
-// cxx11-warning@5 {{use of the 'nodiscard' attribute is a C++17 extension}}
-// cxx11-warning@9 {{use of the 'nodiscard' attribute is a C++17 extension}}
-// cxx11-warning@12 {{use of the 'nodiscard' attribute is a C++17 extension}}
-// cxx11-warning@13 {{use of the 'nodiscard' attribute is a C++17 extension}}
-// cxx11-warning@29 {{use of the 'nodiscard' attribute is a C++17 extension}}
-// cxx11-warning@65 {{use of the 'nodiscard' attribute is a C++20 extension}}
-// cxx11-warning@67 {{use of the 'nodiscard' attribute is a C++20 extension}}
-// cxx11-warning@71 {{use of the 'nodiscard' attribute is a C++20 extension}}
-// cxx11-warning@73 {{use of the 'nodiscard' attribute is a C++20 extension}}
-// cxx11-warning@74 {{use of the 'nodiscard' attribute is a C++20 extension}}
-// cxx11-warning@84 {{use of the 'nodiscard' attribute is a C++20 extension}}
-// cxx11-warning@86 {{use of the 'nodiscard' attribute is a C++17 extension}}
-// cxx11-warning@87 {{use of the 'nodiscard' attribute is a C++20 extension}}
-// cxx11-warning@91 {{use of the 'nodiscard' attribute is a C++17 extension}}
-// cxx11-warning@92 {{use of the 'nodiscard' attribute is a C++20 extension}}
-// cxx11-warning@95 {{use of the 'nodiscard' attribute is a C++20 extension}}
```

Comment:
I will say, this was very confusing to me just now. Are there any actual changes left in this file at this point? This refactoring here is nice, but it should be part of a separate NFC patch if there are no other changes in this file...